sample_data/users.json

```diff
@@ -47,30 +47,6 @@
     "createdAt": "2023-04-13T04:53:17.742Z",
     "__v": 0
   },
-  {
-    "_id": "66378abd85008f171cf2990d",
-    "tokenVersion": 0,
-    "appLanguageCode": "en",
-    "createdOrganizations": [],
-    "createdEvents": [],
-    "userType": "ADMIN",
-    "joinedOrganizations": ["6637904485008f171cf29924"],
-    "registeredEvents": [],
-    "eventAdmin": [],
-    "adminFor": ["6637904485008f171cf29924"],
```

Comment:
1. Why was this removed?
2. Please restore
2. We don't want any existing functionality to be changed in importing sample data